packages/amazon-sumerian-hosts-babylon/examples/gestures-demo.js

```diff
@@ -0,0 +1,79 @@
+import DemoUtils from './common/demo-utils.js';
+import cognitoIdentityPoolId from './common/demo-credentials.js';
+
+let host;
+let scene;
+
+async function createScene() {
+  // Create an empty scene. IMPORTANT: Sumerian Hosts require use of the
+  // right-hand coordinate system!
+  scene = new BABYLON.Scene();
+  scene.useRightHandedSystem = true; // IMPORTANT for Sumerian Hosts!
+
+  DemoUtils.setupSceneEnvironment(scene);
+  initUi();
+
+  // ===== Configure the AWS SDK =====
+
+  AWS.config.region = cognitoIdentityPoolId.split(':')[0];
+  AWS.config.credentials = new AWS.CognitoIdentityCredentials(
+    { IdentityPoolId: cognitoIdentityPoolId },
+  );
+
+  // ===== Instantiate the Sumerian Host =====
+
+  // Edit the characterId if you would like to use one of
+  // the other pre-built host characters. Available character IDs are:
+  // "Cristine", "Fiona", "Grace", "Maya", "Jay", "Luke", "Preston", "Wes"
+  const characterId = 'Cristine';
+  const characterConfig = HOST.HostUtils.getCharacterConfig('./assets/character-assets', characterId);
+  const pollyConfig = { pollyVoice: 'Joanna', pollyEngine: 'neural' };
+  host = await HOST.HostUtils.createHost(scene, characterConfig, pollyConfig);
+
+  // Tell the host to always look at the camera.
+  const camera = scene.cameras[0];
+  host.PointOfInterestFeature.setTarget(camera);
+
+  // Enable shadows.
+  const light = scene.lights[0];
+  const shadowGenerator = new BABYLON.ShadowGenerator(2048, light);
```

Comment:
nit: do we want to use shadowGenerator returned by DemoUtils.setupSceneEnvironment

Reply:
For now I'm choosing to use it in some demos and not others. My goal is that each demo should be slimmed down as much as possible to keep the user's focus on the specific feature we're trying to demonstrate. I don't consider shadows critical to showing off the gestures feature. I will leave this as-is for now.

Reply:
Ah. I now understand your comment better. In my next update to this PR you'll see I've updated the code to use the shadowGenerator returned by setupSceneEnvironment(). Great catch!

Reply:
This issue is addressed in the update I just posted.